STCore/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/managers/TransferManager.kt

```diff
@@ -366,6 +366,29 @@ class TransferManager internal constructor(
         transferController.deleteTransfer(transferUUID)
     }
 
+    /**
+     * Delete a transfer by deeplink.
+     *
+     * @param transferUUID The UUID of the transfer to be removed.
+     * @param token Delete token.
+     *
+     * @throws CancellationException If the operation is cancelled.
+     * @throws ApiErrorException If there is an error related to the API during transfer retrieval.
+     * @throws UnexpectedApiErrorFormatException Unparsable api error response.
+     * @throws NetworkException If there is a network issue during the transfer retrieval.
+     * @throws UnknownException Any error not already handled by the above ones.
+     */
+    @Throws(
+        CancellationException::class,
+        ApiErrorException::class,
+        UnexpectedApiErrorFormatException::class,
+        NetworkException::class,
+        UnknownException::class,
+    )
+    suspend fun deleteTransfer(transferUUID: String, token: String): Unit = withContext(Dispatchers.Default) {
+        transferRepository.delete(transferUUID, token)
```

Comment:
You must also delete the local transfer if it exists, after deleting it remotely.